### PR TITLE
Docs formatting, some list reordering

### PR DESCRIPTION
**Pull request by petecooper**
Docs formatting and presentation to max 80 characters with no widowed words.

No new code,  no removals. Some minor reordering on config arrays (e.g. media types) to be alphabetical, but only where it's clear they _want_ to be alphabetical.

### CODE REVIEW

File: conf.d/.default.conf [hunk 1]
```diff
@@ -7,7 +7,7 @@
 # If no default server is defined, Nginx will use the first found server.
 # To prevent host header attacks, or other potential problems when an unknown
 # servername is used in a request, it's recommended to drop the request
-# returning 444 "no response".
+# returning 444 "No Response".
```
Comment (LeoColomb) [inline conf.d/.default.conf]: 👍

File: conf.d/no-ssl.default.conf [hunk 2]
```diff
@@ -6,8 +6,8 @@
 #
 # If no default server is defined, Nginx will use the first found server.
 # To prevent host header attacks, or other potential problems when an unknown
-# servername is used in a request, it's recommended to drop the request
-# returning 444 "no response".
+# `server_name` is used in a request, it's recommended to drop the request
```
Comment (LeoColomb) [inline conf.d/no-ssl.default.conf]: Nope, this refers to a variable, actually `$hostname` (or whichever used for server selection).
Maybe add a space for correctness, but let keep "server name" reference for "comprehensibility".

```suggestion
# server name is used in a request, it's recommended to drop the request
```

File: h5bp/ssl/policy_modern.conf [hunk 3]
```diff
@@ -26,8 +26,8 @@
 #     $ssl_early_data variable should be used:
 #       proxy_set_header Early-Data $ssl_early_data;
 #
-#     The application should return response code 425 for anything that
-#     could contain user supplied data.
+#     The application should return response code 425 ("Too Early") for anything
```
Comment (LeoColomb) [inline h5bp/ssl/policy_modern.conf]:
```suggestion
#     The application should return response code 425 "Too Early" for anything
```

File: mime.types [hunk 4]
```diff
@@ -122,14 +122,14 @@ types {
     application/xhtml+xml                 xhtml;
     application/xslt+xml                  xsl;
     application/zip                       zip;
+    text/calendar                         ics;
```
Comment (LeoColomb) [inline mime.types]: Indeed 👍

File: conf.d/no-ssl.default.conf (hunk 2 above)
Comment (petecooper) [reply conf.d/no-ssl.default.conf]: Changed in https://github.com/h5bp/server-configs-nginx/pull/232/commits/086207053847e6d952981a77659135b9d4f324e0

File: h5bp/ssl/policy_modern.conf (hunk 3 above)
Comment (petecooper) [reply h5bp/ssl/policy_modern.conf]: Fixed in https://github.com/h5bp/server-configs-nginx/pull/232/commits/863c0fb9bbcd16636096c3d27c594e20a769668a

**Comment by petecooper** [comment]
@LeoColomb - thank you for the review, I appreciate it! The issues you mentioned have been resolved, please advise if anything else needs to be rectified. If you are able to rebase, that would be helpful - I have not done that on a high profile repo before.

**Comment by LeoColomb** [comment]
Merged as 67c54c53f16ccff8aca17aebad9325127f721700.

**Comment by petecooper** [comment]
@nurrabazo –

>I want to know who is the original requester on this thread of 232

That's me.

>What it looks like is you just help these people steal 100 million dollars from me!!!

No, I updated some documentation.

You're in the wrong place, take it elsewhere.

**Pull request closed without merging**